### DISCUSSION TITLE
Organization title truncated in search results

### DISCUSSION THREAD

**Discussion by IgnazNepumuk** [dani-garcia/vaultwarden#4232] [Ideas]
We have multiple Organizations with similar names.

When you search for an entry and results exist in multiple organization its almost impossible to tell to which organization the result belongs to.

In the results view the  long organization name gets truncated ("longOrgaNa...") and even the tooltip  has the same truncated text.

It would be good if at least the tool tip would show the full name of that organization

**Comment by BlackDex** [comment]
That is a web-vault interface item. That is not something the Vaultwarden project maintains. If you can replicate this also on vault.bitwarden.com, please report this over there if that is not already done.

**Comment by IgnazNepumuk** [comment]
the problem is with free accounts you can only have 1 organization. So I really can't test this on bitwarden
